remote/bitbucketserver/bitbucketserver.go

```diff
@@ -115,16 +116,24 @@ func (c *client) Login(res http.ResponseWriter, req *http.Request) (*model.User,
 
 	// TODO errors should never be ignored like this
 	response1, err := client.Get(fmt.Sprintf("%s/rest/api/1.0/users/%s", c.URL, login))
+	if err != nil {
+		return nil, err
+	}
 	contents, err := ioutil.ReadAll(response1.Body)
+	if err !=nil {
+		return nil, err
```

Comment:
if we get an error here, the response won't be closed and will result in open file descriptors. We should add the defer statement right after the response block, before attempting to read